integration/clean/clean_aws_resource.go

```diff
@@ -0,0 +1,145 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+//go:build clean
+// +build clean
+
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	ec2Type "github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	ssmType "github.com/aws/aws-sdk-go-v2/service/ssm/types"
+	smithyTime "github.com/aws/smithy-go/time"
+)
+
+const (
+	daysToKeep   = 30
+	keepDuration = -1 * time.Hour * 24 * time.Duration(daysToKeep)
+)
+
+var expirationDate = time.Now().UTC().Add(keepDuration)
+
+func main() {
+	log.Println("Begin to clean EC2 AMI")
```

Comment:
I think that if we wanted to print when we're starting an action, you could just put it at the beginning of the `cleanAMI()` function itself. Otherwise, as we add more things to clean up here, we are forced to add more prints to this main function.

Reply:
I'm good with either way since it serves the same purpose to me.